tests/main.js

```diff
@@ -1020,6 +1020,73 @@ testBundlers(
   },
 )
 
+testBundlers(
+  'Places all user-defined files at the root of the target directory if the `experimentalHandlerV2` configuration property is defined',
+  [ESBUILD, ESBUILD_ZISI, DEFAULT],
+  async (bundler, t) => {
+    const fixtureName = 'base_path'
+    const { tmpDir } = await zipNode(t, `${fixtureName}/netlify/functions1`, {
+      opts: {
+        basePath: join(FIXTURES_DIR, fixtureName),
+        config: {
+          '*': {
+            experimentalHandlerV2: true,
```

Comment:
Should we add a test checking that `experimentalHandlerV2` defaults to `false` and has the impact of using the old mechanism?

Reply:
Definitely! Done in https://github.com/netlify/zip-it-and-ship-it/pull/474/commits/99b4ff8770c8e35fc68c7b49ce7b83eccae23657.